README.md

```diff
@@ -1,264 +1,8 @@
-goal is to apply them to all of CFPB projects.
-
-We are proud to join the community of organizations that have made their
-design standards public, such as
-[Mozilla](http://www.mozilla.org/en-US/styleguide/),
-[BBC](http://www.bbc.co.uk/gel), and the UK's
-[Government Digital Service](https://www.gov.uk/service-manual).
-We hope our design manual can serve as a foundation for discussing and
-practicing user-centered design in government.
-
-All content has been released as open source under the
-CC0 1.0 Universal Public Domain Dedication, and we'd love for other agencies,
-developers, or groups to adapt it for their own use.
-
-
-## Technology stack
-
-- [Ruby](https://www.ruby-lang.org/en/) - For installing Jekyll.
-- [Jekyll](http://jekyllrb.com/) - Static site server.
-- [Node](https://nodejs.org/en/) - For managing front-end dependencies.
-- [Grunt](http://gruntjs.com/) - Task runner for pulling in assets,
-  linting and concatenating code, etc.
-- [Less](http://lesscss.org/) - CSS pre-processor.
-- [Capital Framework](https://cfpb.github.io/capital-framework/) -
-  User interface pattern-library produced by the CFPB.
-
-**NOTE:** If you're new to Capital Framework, we encourage you to
-[start here](https://cfpb.github.io/capital-framework/getting-started).
-
-
-## Installation
-
-This site is powered by Jekyll a Ruby based static site generator. For
-front-end tooling and asset management we use Node and Grunt. Before
-running the site locally you will need these dependencies. We use
-[homebrew](http://brew.sh/) on Mac OSX to manage installation of software.
-To install the project dependencies using homebrew enter the following:
-
-```shell
-brew install ruby
-brew install node
-gem install jekyll
-npm install --global grunt-cli
-```
-
-To install the site's dependencies, navigate to the project directory and run:
-
-```shell
-./setup.sh
-```
-
-If you receive an error saying that `bundle: command not found`, run `bundle install` before re-running `./setup.sh`.
-
-To launch the site, enter:
-
-```shell
-npm start
-```
-
-This will start the Jekyll server and the Grunt watch task. Open a browser
-window at http://localhost:4000.
-
-### Developing
-
-When first setting up this project, and each time you fetch from upstream,
-run the setup shell script to install the newest project dependencies and
-build the website with Grunt:
-
-```sh
-./setup.sh
-```
-
-We use Grunt to build our asset files. The easiest way to do that is to run
-the `watch` task which will monitor file changes and re-build the assets
-whenever you save a CSS or JS file
-(**Note: This is running when you run `npm start`, no need to run it twice**):
-
-```
-grunt watch
-```
-
-
-## _config.yml
-
-Options within the `_config.yml` file allow you to control the site's title,
-subtitle, logo, author information, and the left column navigation.
+# CFPB Design Manual
 
-### Project Page URL Structure
+# :warning: THIS REPO IS DEPRECATED :warning:
```

Comment:
Maybe add a date stamp. I guess that's dictated by the commit stamp, but like https://github.com/cfpb/cf-buttons#warning-this-repo-is-deprecated-1152016-warning can be nice to see at a glance.